examples/simple_solution.xml

```diff
@@ -0,0 +1,99 @@
+<database
+        xmlns="http://calphad.org/thermml/0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:xi="http://www.w3.org/2001/XInclude"
+        xsi:schemaLocation="http://calphad.org/thermml/0.0 ../schema/thermml-schema.xsd"
+
+        name="Example Simple Solution Database">
+
+    <description>
+        This is an example thermodynamic database for demonstrating the ThermML schema for simple solutions.
+    </description>
+
+    <nonChemicalPotentials>
```

Comment:
We use the name `IndependentPotential` in pycalphad for this idea.

Reply:
Not really (yet). In principle a (for example) magnetic field-dependent database would work with no modifications in PyCalphad today because of our use of `IndependentPotential` to define custom variables, but there is no nice example yet. It is mostly used today for calculating Jansson derivatives with respect to model parameters (which can formally be treated the same as potentials for calculation purposes).